### ISSUE TITLE
[A11Y] Missing locale tag parameter for each language on the switch locale at the right corner top

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#9343]
## Issue Description
Assistive Technologies (AT) as screenreaders will announce language names on the language declared for the page instead of reading on the language pronounciation itself.

## How to reproduce / Current behavior
1. Log in to the journal
2. Activate other locales to the journal
3. Click/activate the profile menu at top right corner
4. The menu shows the language without locale parameter

## What application are you using?
 OJS version 3.4.0

## Stack used 
- Device: Laptop
- OS: MacOS(Monterey)
- Browser: Chrome
- Version: 117

## Testing method
- Method: Screen Reader (VoiceOver)

## Action
Include in the A HREF tag the parameter " lang=" and the value for each aviallable locale to journal/website. 

## Additional information
![Image](https://github.com/pkp/pkp-lib/assets/4992902/9ab5c581-d400-40c6-a5a3-84283d23321c)
